broker/lib/fabrik/DBManager.js

```diff
@@ -251,6 +287,21 @@ class DBManager {
       });
   }
 
+  storeBindPropertyOnApiServer(bindProperty) {
+    return eventmesh.apiServerClient.createResource({
+      resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BIND,
+      resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR_BIND,
+      resourceId: bindProperty.id,
+      options: {
+        binding_id: bindProperty.id
+      },
+      status: {
+        state: CONST.APISERVER.RESOURCE_STATE.SUCCEEDED,
+        response: bindProperty
```

Comment:
We usually store [base64 encoded](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/operators/bosh-operator/BoshBindOperator.js#L60) credentials in binding resources.
Is it possible to do same here?

Reply:
Yes, will modify. I had missed this. Thanks for pointing out.

Reply:
Done.